### DISCUSSION TITLE
https://johanneskonings.dev/notes/2024-06-16-cdk-nag/

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#26] [Note comments]
# https://johanneskonings.dev/notes/2024-06-16-cdk-nag/

https://johanneskonings.dev/notes/2024-06-16-cdk-nag/

**Comment by JohannesKonings** [comment]
https://docs.aws.amazon.com/cdk/v2/guide/policy-validation-synthesis.html

**Comment by JohannesKonings** [comment]
https://github.com/awslabs/iac-devtools-cli-for-cdk

**Comment by JohannesKonings** [comment]
tags alarm:
* https://github.com/aws/aws-cdk/issues/29759
* https://github.com/aws/aws-cdk/issues/25489#issue-1701033601
